webapp/src/main/java/io/github/microcks/util/grpc/GrpcTestRunner.java

```diff
@@ -60,6 +70,36 @@
  */
 public class GrpcTestRunner extends AbstractTestRunner<HttpMethod> {
 
+   /* Call Option used to pass gRPC Metadata from client invocation to header client interceptor */
+   public static final CallOptions.Key<Metadata> METADATA_CUSTOM_CALL_OPTION = CallOptions.Key
+         .createWithDefault("request-metadata", null);
```

Comment:
Is `request-metadata` a standard stuff? Don't we have a Java constant for that in the `grpc-java` library?

Reply:
No there is no standard. Actually any string can be used as a name here. It should just describe the key. I moved it into a constant to highlight the intention of the name.